src/styles/webm.css

```diff
@@ -1,24 +1,27 @@
 /* Custom styles for non-Starlight pages */
+/* @link https://utopia.fyi/type/calculator?c=320,16,1.125,1440,16,1.2,6,2,&s=0.75|0.5|0.25,1.5|2|3|4|6,s-l&g=s,l,xl,12 */
 :root {
-  --step--2: clamp(0.69rem, calc(0.82rem + -0.14vw), 0.79rem);
-  --step--1: clamp(0.83rem, calc(0.9rem + -0.08vw), 0.89rem);
-  --step-0: clamp(1rem, calc(1rem + 0vw), 1rem);
-  --step-1: clamp(1.13rem, calc(1.1rem + 0.11vw), 1.2rem);
-  --step-2: clamp(1.27rem, calc(1.22rem + 0.25vw), 1.44rem);
-  --step-3: clamp(1.42rem, calc(1.34rem + 0.43vw), 1.73rem);
-  --step-4: clamp(1.6rem, calc(1.47rem + 0.67vw), 2.07rem);
-  --step-5: clamp(1.8rem, calc(1.61rem + 0.98vw), 2.49rem);
-  --space-3xs: clamp(0.25rem, calc(0.25rem + 0vw), 0.25rem);
-  --space-2xs: clamp(0.5rem, calc(0.5rem + 0vw), 0.5rem);
-  --space-xs: clamp(0.75rem, calc(0.75rem + 0vw), 0.75rem);
-  --space-s: clamp(1rem, calc(1rem + 0vw), 1rem);
-  --space-m: clamp(1.5rem, calc(1.5rem + 0vw), 1.5rem);
-  --space-l: clamp(2rem, calc(2rem + 0vw), 2rem);
-  --space-xl: clamp(3rem, calc(3rem + 0vw), 3rem);
-  --space-2xl: clamp(4rem, calc(4rem + 0vw), 4rem);
-  --space-3xl: clamp(6rem, calc(6rem + 0vw), 6rem);
+  --step--2: clamp(0.6944rem, 0.8175rem + -0.1367vi, 0.7901rem);
```

Comment:
Utopia.fyi updated their clamp calculations, also added the permalink so I don't have to keep resizing the browser to discover what my original options were.